no_json_logging.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2017-2024 Fortio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Moved json logging out so it can be skipped for smallest binaries based on build tags.
```

Comment:
```suggestion
// Moved JSON logging out so it can be skipped for smallest binaries based on build tags.
```

Reply:
thank! any other? I guess you should make your tool into a ci linter :)

Reply:
I did some git grep + search and replace for some/most/many json->JSON

Reply:
> thank! any other? I guess you should make your tool into a ci linter :)

It's planned. Please star and watch my repository

https://github.com/jargonLint/jargonLint